docs/user_guide/Upgrade_SIMP/Version_Maps/6.3.3_6.4.0.inc

```diff
@@ -71,3 +79,152 @@ functionality:
    for x in *.rpmsave; do
      mv $x `basename $x .rpmsave`
    done
+
+Upgrade Steps
+^^^^^^^^^^^^^
+These steps for upgrading are for use in SIMP systems that are not currently using
+r10k or Code Manager.
+
+Update OS and SIMP RPMS
+"""""""""""""""""""""""
+
+Update the repos for the OS and SIMP and run yum update
+
+The following errors/warnings are expected:
+
+.. code-block:: bash
+
+  warning: /var/simp/environments/simp/FakeCA/togen saved as /var/simp/environments/simp/FakeCA/togen.rpmsave
+  warning: /var/simp/environments/simp/FakeCA/ca.cnf saved as /var/simp/environments/simp/FakeCA/ca.cnf.rpmsave
+  libsemanage.semanage_direct_remove_key: Unable to remove module simp-environment at priority 400. (No such file or directory).
+  /usr/sbin/semodule:  Failed!
+
+If you are using the ``FakeCA`` in the ``simp`` environment, follow the steps in the ``FakeCA`` section above to restore the needed files.
+
+The old selinux module, ``simp-environment``, had to be removed before installing the
```

Comment:
Should this description be after the warning so that its context makes more sense?